Build Problems

Hi guys, It's been a few years so I figured Id take a look at how things are progressing. I don't really have the time to do anything, but that doesn't mean I'm disinterested!

Following the readme on how to build with cmake, I get the following error(s):

```
F:\Code\new anh\swganh\build>cmake --build .

Microsoft (R) Visual Studio 11 Version 11.0.50323.1.
Copyright (C) Microsoft Corp. All rights reserved.

The evaluation period for Visual Studio Trial ends in 205 days.
1>------ Build started: Project: ZERO_CHECK, Configuration: Debug Win32 ------
1>  Checking Build System
1>  CMake does not need to re-run because F:/Code/new anh/swganh/build/CMakeFile
s/generate.stamp is up-to-date.
1>  CMake does not need to re-run because F:/Code/new anh/swganh/build/src/CMake
Files/generate.stamp is up-to-date.
1>  CMake does not need to re-run because F:/Code/new anh/swganh/build/src/anh/C
MakeFiles/generate.stamp is up-to-date.
1>  CMake does not need to re-run because F:/Code/new anh/swganh/build/src/swgan
h/CMakeFiles/generate.stamp is up-to-date.
1>  CMake does not need to re-run because F:/Code/new anh/swganh/build/src/swgan
h/app/CMakeFiles/generate.stamp is up-to-date.
1>  CMake does not need to re-run because F:/Code/new anh/swganh/build/src/swgan
h/command/CMakeFiles/generate.stamp is up-to-date.
1>  CMake does not need to re-run because F:/Code/new anh/swganh/build/src/swgan
h/combat/CMakeFiles/generate.stamp is up-to-date.
1>  CMake does not need to re-run because F:/Code/new anh/swganh/build/src/swgan
h/messages/CMakeFiles/generate.stamp is up-to-date.
1>  CMake does not need to re-run because F:/Code/new anh/swganh/build/src/swgan
h/object/CMakeFiles/generate.stamp is up-to-date.
1>  CMake does not need to re-run because F:/Code/new anh/swganh/build/src/swgan
h/social/CMakeFiles/generate.stamp is up-to-date.
1>  CMake does not need to re-run because F:/Code/new anh/swganh/build/src/swgan
h/simulation/CMakeFiles/generate.stamp is up-to-date.
1>  CMake does not need to re-run because F:/Code/new anh/swganh/build/src/swgan
h/utilities/CMakeFiles/generate.stamp is up-to-date.
1>  CMake does not need to re-run because F:/Code/new anh/swganh/build/src/pub14
_core/CMakeFiles/generate.stamp is up-to-date.
1>  CMake does not need to re-run because F:/Code/new anh/swganh/build/src/swged
/CMakeFiles/generate.stamp is up-to-date.
2>------ Build started: Project: anh_lib, Configuration: Debug Win32 ------
3>------ Build started: Project: DEPS, Configuration: Debug Win32 ------
2>  Building Custom Rule F:/Code/new anh/swganh/src/anh/CMakeLists.txt
2>  CMake does not need to re-run because F:\Code\new anh\swganh\build\src\anh\C
MakeFiles\generate.stamp is up-to-date.
2>  byte_buffer.cc
3>  Building Custom Rule F:/Code/new anh/swganh/src/CMakeLists.txt
3>  CMake does not need to re-run because F:\Code\new anh\swganh\build\src\CMake
Files\generate.stamp is up-to-date.
3>  F:\Code\new anh\swganh\vendor\boost\lib\boost_date_time-vc110-mt-gd-1_49.dll

3>  F:\Code\new anh\swganh\vendor\boost\lib\boost_filesystem-vc110-mt-gd-1_49.dl
l
3>  F:\Code\new anh\swganh\vendor\boost\lib\boost_log-vc110-mt-gd-1_49.dll
3>  F:\Code\new anh\swganh\vendor\boost\lib\boost_log_setup-vc110-mt-gd-1_49.dll

3>  F:\Code\new anh\swganh\vendor\boost\lib\boost_prg_exec_monitor-vc110-mt-gd-1
_49.dll
3>  F:\Code\new anh\swganh\vendor\boost\lib\boost_program_options-vc110-mt-gd-1_
49.dll
3>  F:\Code\new anh\swganh\vendor\boost\lib\boost_python-vc110-mt-gd-1_49.dll
3>  F:\Code\new anh\swganh\vendor\boost\lib\boost_python3-vc110-mt-gd-1_49.dll
3>  F:\Code\new anh\swganh\vendor\boost\lib\boost_random-vc110-mt-gd-1_49.dll
3>  F:\Code\new anh\swganh\vendor\boost\lib\boost_regex-vc110-mt-gd-1_49.dll
3>  F:\Code\new anh\swganh\vendor\boost\lib\boost_system-vc110-mt-gd-1_49.dll
3>  F:\Code\new anh\swganh\vendor\boost\lib\boost_thread-vc110-mt-gd-1_49.dll
3>  F:\Code\new anh\swganh\vendor\boost\lib\boost_timer-vc110-mt-gd-1_49.dll
3>  F:\Code\new anh\swganh\vendor\boost\lib\boost_unit_test_framework-vc110-mt-g
d-1_49.dll
3>  14 File(s) copied
3>  F:\Code\new anh\swganh\vendor\boost\lib\boost_date_time-vc110-mt-1_49.dll
3>  F:\Code\new anh\swganh\vendor\boost\lib\boost_filesystem-vc110-mt-1_49.dll
3>  F:\Code\new anh\swganh\vendor\boost\lib\boost_log-vc110-mt-1_49.dll
3>  F:\Code\new anh\swganh\vendor\boost\lib\boost_log_setup-vc110-mt-1_49.dll
3>  F:\Code\new anh\swganh\vendor\boost\lib\boost_prg_exec_monitor-vc110-mt-1_49
.dll
3>  F:\Code\new anh\swganh\vendor\boost\lib\boost_program_options-vc110-mt-1_49.
dll
3>  F:\Code\new anh\swganh\vendor\boost\lib\boost_python-vc110-mt-1_49.dll
3>  F:\Code\new anh\swganh\vendor\boost\lib\boost_python3-vc110-mt-1_49.dll
3>  F:\Code\new anh\swganh\vendor\boost\lib\boost_random-vc110-mt-1_49.dll
3>  F:\Code\new anh\swganh\vendor\boost\lib\boost_regex-vc110-mt-1_49.dll
3>  F:\Code\new anh\swganh\vendor\boost\lib\boost_system-vc110-mt-1_49.dll
3>  F:\Code\new anh\swganh\vendor\boost\lib\boost_thread-vc110-mt-1_49.dll
3>  F:\Code\new anh\swganh\vendor\boost\lib\boost_timer-vc110-mt-1_49.dll
3>  F:\Code\new anh\swganh\vendor\boost\lib\boost_unit_test_framework-vc110-mt-1
_49.dll
3>  14 File(s) copied
3>  F:\Code\new anh\swganh\vendor\mysql-connector-c\lib\opt\..\Debug\libmysql.dl
l
3>  1 File(s) copied
3>  F:\Code\new anh\swganh\vendor\mysql-connector-c\lib\opt\..\opt\libmysql.dll
3>  1 File(s) copied
3>  F:\Code\new anh\swganh\vendor\mysql-connector-cpp\lib\Debug\..\Debug\mysqlcp
pconn.dll
3>  1 File(s) copied
3>  C:\Python32\libs\..\DLLs\python3.dll
3>  1 File(s) copied
3>  F:\Code\new anh\swganh\vendor\zlib\lib\Debug\..\Debug\zlibd.dll
3>  F:\Code\new anh\swganh\vendor\zlib\lib\Debug\..\Debug\zlibd1.dll
3>  2 File(s) copied
3>  Invalid number of parameters
3>  0 File(s) copied
3>  Invalid number of parameters
3>  0 File(s) copied
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: The command "setlocal
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: xcopy "F:\Code\new anh\swganh\vendor\boost\lib\*-gd-
*.dll" "F:\Code\new anh\swganh\build\bin\Debug" /D /I /Y /s
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: if %errorlevel% neq 0 goto :cmEnd
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: xcopy "F:\Code\new anh\swganh\vendor\boost\lib\*-mt-
1*.dll" "F:\Code\new anh\swganh\build\bin\Release" /D /I /Y /s
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: if %errorlevel% neq 0 goto :cmEnd
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: :cmEnd
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: endlocal & call :cmErrorLevel %errorlevel% & goto :c
mDone
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: :cmErrorLevel
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: exit /b %1
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: :cmDone
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: if %errorlevel% neq 0 goto :VCEnd
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: setlocal
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: xcopy "F:\Code\new anh\swganh\vendor\mysql-connector
-c\lib\opt\..\Debug\*.dll" "F:\Code\new anh\swganh\build\bin\Debug" /D /I /Y /s
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: if %errorlevel% neq 0 goto :cmEnd
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: xcopy "F:\Code\new anh\swganh\vendor\mysql-connector
-c\lib\opt\..\opt\*.dll" "F:\Code\new anh\swganh\build\bin\Debug" /D /I /Y /s
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: if %errorlevel% neq 0 goto :cmEnd
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: :cmEnd
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: endlocal & call :cmErrorLevel %errorlevel% & goto :c
mDone
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: :cmErrorLevel
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: exit /b %1
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: :cmDone
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: if %errorlevel% neq 0 goto :VCEnd
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: setlocal
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: xcopy "F:\Code\new anh\swganh\vendor\mysql-connector
-cpp\lib\Debug\..\Debug\*.dll" "F:\Code\new anh\swganh\build\bin\Debug" /D /I /Y
 /s
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: if %errorlevel% neq 0 goto :cmEnd
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: :cmEnd
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: endlocal & call :cmErrorLevel %errorlevel% & goto :c
mDone
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: :cmErrorLevel
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: exit /b %1
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: :cmDone
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: if %errorlevel% neq 0 goto :VCEnd
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: setlocal
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: xcopy C:\Python32\libs\..\DLLs\python3.dll "F:\Code\
new anh\swganh\build\bin\Debug" /D /I /Y /s
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: if %errorlevel% neq 0 goto :cmEnd
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: :cmEnd
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: endlocal & call :cmErrorLevel %errorlevel% & goto :c
mDone
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: :cmErrorLevel
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: exit /b %1
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: :cmDone
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: if %errorlevel% neq 0 goto :VCEnd
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: setlocal
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: xcopy "F:\Code\new anh\swganh\vendor\zlib\lib\Debug\
..\Debug\*.dll" "F:\Code\new anh\swganh\build\bin\Debug" /D /I /Y /s
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: if %errorlevel% neq 0 goto :cmEnd
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: :cmEnd
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: endlocal & call :cmErrorLevel %errorlevel% & goto :c
mDone
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: :cmErrorLevel
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: exit /b %1
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: :cmDone
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: if %errorlevel% neq 0 goto :VCEnd
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: setlocal
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: for /R "F:\Code\new anh\swganh\plugins\" %%a IN (*.c
fg) do (xcopy %%a "F:\Code\new anh\swganh\build\bin\Debug\config\plugins\" /D /I
 /Y)
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: if %errorlevel% neq 0 goto :cmEnd
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: :cmEnd
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: endlocal & call :cmErrorLevel %errorlevel% & goto :c
mDone
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: :cmErrorLevel
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: exit /b %1
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: :cmDone
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: if %errorlevel% neq 0 goto :VCEnd
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: setlocal
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: for /R "F:\Code\new anh\swganh\src\" %%a IN (*.cfg)
do (xcopy %%a "F:\Code\new anh\swganh\build\bin\Debug\config\plugins\" /D /I /Y)

3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: if %errorlevel% neq 0 goto :cmEnd
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: :cmEnd
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: endlocal & call :cmErrorLevel %errorlevel% & goto :c
mDone
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: :cmErrorLevel
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: exit /b %1
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: :cmDone
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: if %errorlevel% neq 0 goto :VCEnd
3>C:\Program Files (x86)\MSBuild\Microsoft.Cpp\v4.0\V110\Microsoft.CppCommon.tar
gets(125,5): error MSB3073: :VCEnd" exited with code 4.
2>  crc.cc
2>  database_manager.cc
2>  event_dispatcher.cc
2>  hash_string.cc
2>  compression_filter.cc
2>  crc_in_filter.cc
2>  crc_out_filter.cc

```

--- a/tools/cmake/Win32PostBuild.cmake
+++ b/tools/cmake/Win32PostBuild.cmake
@@ -70,11 +70,11 @@ endforeach()
 add_custom_command(
     TARGET DEPS
     POST_BUILD
-    COMMAND for /R \"${WIN_PROJECT_SOURCE_DIR}\\plugins\\\" %%a IN \(*.cfg\) do \(xcopy %%a \"${WIN_PROJECT_BINARY_DIR}\\bin\\$\(Configuration\)\\config\\plugins\\\" /D /I /Y\)
+    COMMAND for /R \"${WIN_PROJECT_SOURCE_DIR}\\plugins\\\" %%a IN \(*.cfg\) do \(xcopy \"%%a\" \"${WIN_PROJECT_BINARY_DIR}\\bin\\$\(Configuration\)\\config\\plugins\\\" /D /I /Y\)
 )
 
 add_custom_command(
     TARGET DEPS
     POST_BUILD
-    COMMAND for /R \"${WIN_PROJECT_SOURCE_DIR}\\src\\\" %%a IN \(*.cfg\) do \(xcopy %%a \"${WIN_PROJECT_BINARY_DIR}\\bin\\$\(Configuration\)\\config\\plugins\\\" /D /I /Y\)
+    COMMAND for /R \"${WIN_PROJECT_SOURCE_DIR}\\src\\\" %%a IN \(*.cfg\) do \(xcopy \"%%a\" \"${WIN_PROJECT_BINARY_DIR}\\bin\\$\(Configuration\)\\config\\plugins\\\" /D /I /Y\)
 )

